packages/desktop-client/src/components/mobile/budget/BudgetTable.jsx

```diff
@@ -61,7 +61,7 @@ export const PILL_STYLE = {
 };
 
 export function getColumnWidth({
-  show3Cols,
+  show3Cols = true,
```

Comment:
Unless I'm missing something, doesn't this change the default behaviour from falsy to true?

Reply:
Good spot! Updated it here: https://github.com/actualbudget/actual/pull/4605/commits/cd7538e44874b055314e143556cafb0c82d5e1d5